File: backend/auth/src/main/java/com/booking/auth/auth/DTO/ClientDTO.java
package com.booking.auth.auth.DTO;

public class ClientDTO {
    public String cpf;
    public String nome;
    public int saldo_milhas;
    public String email;
    public EnderecoDTO endereco;

    public static class EnderecoDTO {
        public String cep;
        public String uf;
        public String cidade;
        public String bairro;
        public String rua;
        public String numero;
        public String complemento;
    }
}

File: src/app/shared/models/sing/clientDto.ts
export class Endereco {
    constructor(
        public cep: string = '',
        public uf: string = '',
        public cidade: string = '',
        public bairro: string = '',
        public rua: string = '',
        public numero: string = '',
        public complemento: string = ''
    ) {}
}

export class ClientDTO {
    constructor(
        public cpf: string = '',
        public email: string = '',
        public nome: string = '',
        public saldo_milhas: number = 0,
        public endereco: Endereco = new Endereco()
    ) {}
}

